database/scrapers/academic_calendar_scraper.ts

```diff
@@ -0,0 +1,89 @@
+import { DateTime } from "luxon";
+
+import { BaseScraperModule, TaskHandle } from "#commands/db_scrape";
+import { Weekday } from "#enums/weekday";
+import AcademicCalendar from "#models/academic_calendar";
+
+interface AcademicCalendarData {
+  data: {
+    id: number;
+    user_updated: string;
+    date_updated: DateTime;
+    semesterStartDate: DateTime;
+    examSessionStartDate: DateTime;
+    isFirstWeekEven: boolean;
+    examSessionLastDay: DateTime;
```

Comment:
An actual issue: The directus API returns strings, not `DateTime` objects. (in fact you cannot automatically deserialize such objects from JSON)
Even though adonis' orm knows how to cope with strings being passed as DateTimes, you should define those fields as strings (because that's what they actually are) and parse them if needed.
You can use `DateTime.fromISO()` to deserialize these DateTimes.